src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4841indentation/InputFormattedIndentationCorrectWhileDoWhileAndParameter.java

```diff
@@ -0,0 +1,270 @@
+package com.google.checkstyle.test.chapter4formatting.rule4841indentation;
+
+class InputFormattedIndentationCorrectWhileDoWhileAndParameter {
```

Comment:
this file is same as InputIndentationCorrectWhileDoWhileAndParameter, what is a point to have it ?

we have only this file not formatted https://github.com/checkstyle/checkstyle/blob/5472ce3b3127d0c6f69bfa6b94978d1792667fe6/.ci/google-java-format.sh#L52

Reply:
few more files are identical (except file/class name). We do not need such files.

Reply:
removed all other formatted files, now we only have formatted file for ClassWithChainedMethods.java which was grepped